src/mlang/mpp_ir/mpp_ir_to_bir.ml

```diff
@@ -318,9 +327,7 @@ and translate_mpp_stmt (mpp_program : Mpp_ir.mpp_compute list)
                    var_definition =
                      SimpleVar (translate_mpp_expr m_program ctx expr, pos);
                    var_typ = None;
-                   var_io =
-                     (snd (Mir.find_var_definition m_program.program var))
-                       .var_io;
+                   var_io = Mir.Input;
```

Comment:
Isn't this done by `Bir_interpreter.RegularFloatInterpreter.replace_undefined_with_input_variables` within `wrap_m_code_call`?

Reply:
That does more than just changing the IO type. It removes the `var_definition` entirely so that the M does not override the set value in the M++.
I actually think that is not necessary, nor does it follow the current semantics of INTER files, so we should probably discuss it further at some point.

Reply:
Okay. Could you please open an issue on this?

Reply:
Thank you! (#136)